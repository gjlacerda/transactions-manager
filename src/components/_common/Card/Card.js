import styled from 'styled-components'
import { colorWhite } from 'utils/color'
import { borderRadius } from 'utils/border'
import { boxShadow } from 'utils/shadow'
import { marginSize } from 'utils/margin'

export default styled.div`
  background-color: ${colorWhite};
  border-radius: ${borderRadius};
  padding: 30px;
  box-shadow: ${boxShadow};
  margin-bottom: ${marginSize.medium};
`
