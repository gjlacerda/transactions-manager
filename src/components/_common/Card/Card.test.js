import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import Card from './Card'

describe('Card component', () => {
  it('deve ser igual ao snapshot', () => {
    const tree = renderer.create(
      <Card />,
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
