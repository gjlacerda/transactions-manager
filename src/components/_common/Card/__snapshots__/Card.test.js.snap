// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card component deve ser igual ao snapshot 1`] = `
.c0 {
  background-color: #ffffff;
  border-radius: 4px;
  padding: 30px;
  box-shadow: 0 2px 3px rgba(0,0,0,0.09);
  margin: 0 10px 20px 10px;
}

<div
  className="c0"
/>
`;
